README.md

```diff
@@ -1,21 +1,72 @@
 # Kubevirt Velero Plugin
 
-Velero is the preferred solution for OpenShift for backup, disaster recovery
-and migration. Out-of-the-box it allows backing up VMs and DVs in a couple
-limited scenarios and requires jumping through several hoops.
+This repository contains Velero plugins. Thanks to this plugin, [Velero](https://velero.io/) can correctly backup and restore
+VirtualMachines, DataVolumess and other resources managed by [KubeVirt](https://kubevirt.io/) and 
+[CDI](https://github.com/kubevirt/containerized-data-importer/).
 
-The plugin will ensure full support as well as automation of the abovementioned
-hoop-jumping. ;-)
+For more information on Velero check https://velero.io/.
 
 This is still a work in progress, not intended for production use.
 
-## Building the plugins
+## Kinds of Plugins Included
+
+Kubevirt Velero Plugin implements two kinds of plugins:
+- **Backup Item Action** - performs arbitrary logic on individual items prior to storing them in the backup file.
+- **Restore Item Action** - performs arbitrary logic on individual items prior to restoring them in the Kubernetes cluster.
+
+The plugin operates on following resources: DataVolume, PersistentVolumeClaim, Pod, VirtualMachine, VirtualMachineInstance
+ 
+## Compatibility
+
+Plugin versions and respective Velero/Kubevirt/CDI versions that are tested to be compatible.
+
+| Plugin Version  | Velero Version | Kubevirt Version | CDI Version  |
+|-----------------|----------------|------------------|--------------|
+| v0.2.0          | v1.6.x, v1.7.x | v0.48.x          | \>= v1.37.0  |
+
+## Install
+
+To install the plugin check current velero documentation https://velero.io/docs/v1.7/overview-plugins/.
+Below example for kubevirt-velero-plugin version v0.2.0 on Velero 1.7.0
+
+```bash
+velero plugin add quay.io/kubevirt/kubevirt-velero-plugin:v0.2.0
+```
+
+## Backup/Restore Virtual Machines Using the Plugin
 
-To build the plugin's builder, run
+When the plugin is deployed it is already effective. There is nothing special required to make it work.
+### 1. Create a Virtual Machine
 
 ```bash
-$ make build-builder
+kubectl create namespace demo
+kubectl create -f example/datavolume.yaml -n demo
+kubectl create -f example/vm.yaml -n demo
 ```
+Start a VM (wait for condition AgentConnected):
+```bash
+virtctl start example-vm -n demo`
+```
```

Comment:
you can make the example shorter if you add `  running: true` to the vm.yaml (not sure you should care about this)

Reply:
sure, that simplifies instruction